### DISCUSSION TITLE
Vite 6.0.2 is out.

### DISCUSSION THREAD

**Discussion by LuigiNicaPRO** [barrel/shopify-vite#159] [General]
Hello,

Vite 6.0.2 is out. How can we move from Vite 5.4 to Vite 6.0.2
https://github.com/vitejs/vite/releases/tag/v6.0.2

Thank you,
Luigi

**Comment by montalvomiguelo** [comment]
Hi @LuigiNicaPRO, we will add support for Vite 6 in an upcoming release https://github.com/barrel/shopify-vite/pull/161 :)
